docs/3.0.0-beta.x/content-api/api-endpoints.md

```diff
@@ -296,7 +304,17 @@ GET http://localhost:1337/restaurants
     "name": "Restaurant 1",
     "cover": {
       "id": 1,
-      "url": "/uploads/3d89ba92f762433bbb75bbbfd9c13974.png"
+      "name": "image.png",
```

Comment:
This will change when we release the ML. Will need to think about update those :)